docker-compose.yml

```diff
@@ -24,6 +24,20 @@ services:
     depends_on:
       - ldap
 
+  mongodb:
```

Comment:
The xdmod container should now depend on mongodb correct? Perhaps add this container to xdmod `depends`.